src/trixi_include.jl

```diff
@@ -18,7 +18,7 @@ for ODEs, see the "Miscellaneous" section of the
 
 # Examples
 
-```jldoctest
+```@example
 julia> redirect_stdout(devnull) do
```

Comment:
```suggestion
julia> using TrixiBase, Trixi

julia> redirect_stdout(devnull) do
```

To clarify that this uses some code from Trixi.jl?

Reply:
Yes, makes sense.